java.lang.StringIndexOutOfBoundsException when parsing MultiZone/Economizers/Controller.mo

@JayHuLBL : Can you please have a look at what causes this error:
```bash
$ node app.js --log warn -f ~/proj/ldrd/bie/modeling/github/lbl-srg/modelica-buildings/Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/Economizers/Controller.mo -o html
error:   *** Error when parsing /Users/mwetter/proj/ldrd/bie/modeling/github/lbl-srg/modelica-buildings/Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/Economizers/Controller.mo: "java.lang.Exception: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at gov.lbl.parser.Main.moParser(Main.java:130)
	at gov.lbl.parser.Main.main(Main.java:53)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.charAt(String.java:646)
	at gov.lbl.parser.domain.Comment.splitAtComma(Comment.java:428)
	at gov.lbl.parser.domain.Comment.access$400(Comment.java:7)
```

--- a/java/parser/src/main/java/gov/lbl/parser/domain/Comment.java
+++ b/java/parser/src/main/java/gov/lbl/parser/domain/Comment.java
@@ -424,6 +424,10 @@ public class Comment {
 				}
     			strSets.add(str.substring(commaInd.get(commaInd.size()-1)+1, str.length()));
     		}
+    		if (strSets.get(strSets.size()-1).isEmpty()) {
+    			strSets.remove(strSets.size()-1);
+    		}
+    		
     		for (int i=0; i<strSets.size(); i++) {
     			if (strSets.get(i).charAt(strSets.get(i).length()-1) == ' ') {
     				String temStr = strSets.get(i);

